packages/kas/src/__tests__/index_test.js

```diff
@@ -1,5 +1,67 @@
+import _ from "underscore";
+
 import * as KAS from "../index.js";
 
+expect.extend({
+    toHaveGCD(input: [string, string], expected: string) {
+        const [a, b] = input;
+        var actualRep = KAS.parse(a).expr.findGCD(KAS.parse(b).expr).repr();
+        var expectedRep = KAS.parse(expected).expr.repr();
+
+        var message = "(" + a + ").findGCD(" + b + ") = " + expected;
```

Comment:
It doesn't look like you use this message.

Reply:
Thanks. I'll clean this up.